web/pdf_viewer.css

```
@@ -157,10 +158,16 @@
   right: 0;
   bottom: 0;
   background: url("images/loading-icon.gif") center no-repeat;
+  visibility: visible;
+  /* Using a delay with bacground-image doesn't work,
+     consequently, we use the visibility. */
```

Spelling/grammar nits:
```suggestion
  /* Using a delay with background-image doesn't work,
     consequently we use the visibility. */
```